### PR TITLE
Mahmoud/testing

### PR DESCRIPTION
**Pull request by CodeMahmoud**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -160,6 +162,54 @@ const createWindow = (exports.createWindow = () => {
...
+    const rows = size.rows;
+    ptyProcess.resize(cols, rows);
+  });
+
+  require('electron-reload')(__dirname, {
+    electron: path.join(__dirname, '../node_modules', '.bin', 'electron'),
+    awaitWriteFinish: true,
+  });
+
+=======
```
Comment (kiranbanger) [inline src/index.js:211]: i think these are leftover from a merge conflict and should probably be removed?

File: src/index.js [hunk 2]
```diff
@@ -160,6 +162,54 @@ const createWindow = (exports.createWindow = () => {
     createApplicationMenu();
     newWindow = null;
   });
+<<<<<<< HEAD
```
Comment (kiranbanger) [inline src/index.js:165]: left over from a merge conflict?

**Review by lherna05 — APPROVED** [review]

File: src/sample.spec.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import { describe, expect, test } from 'vitest';
```
Comment (kiranbanger) [inline src/sample.spec.js:1]: does this file need to be included? do we have any testing implemented?

File: src/index.js (hunk 1 above)
Comment (CodeMahmoud) [reply src/index.js:211]: I'll clean that out

**Review by kiranbanger — COMMENTED** [review]
i think there's some things that need to removed

File: src/index.js (hunk 2 above)
Comment (CodeMahmoud) [reply src/index.js]: removed line 165

**Review by kiranbanger — APPROVED** [review]
looks good!

**Pull request merged**